CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on the [KeepAChangeLog] project.
 - [#607] Fixed key recovery on encryption of payload
 - [#618] Prettified `client_management.py` CLI and wrapped it as
          a setup.py console script `oic-client-management`
+- [#615] Fir ROPC grant in the extensions client
```

Comment:
Fix ;-)

And isn't it the extensions provider?

Reply:
Fixed